podwatcher/extension_test.go

```diff
@@ -0,0 +1,237 @@
+package podwatcher_test
+
+import (
+	"context"
+	"encoding/json"
+
+	. "code.cloudfoundry.org/eirini-loggregator-bridge/podwatcher"
+	eirinixcatalog "code.cloudfoundry.org/eirinix/testing"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+func decodePatches(resp admission.Response) string {
```

Comment:
This is, then, equivalent to `return strings.Join(jsonifyPatches(resp), "")`?

Reply:
Done in https://github.com/cloudfoundry-incubator/eirini-loggregator-bridge/pull/26/commits/7887894217194e764b8634c7b6857d4e9b915a9b